_drafts/git-bisect.md

```diff
@@ -0,0 +1,585 @@
+---
+title: "Hunting Bugs with Git Bisect"
+description: "An introduction to bisecting a set of changes as a bug-hunting tool, followed by some deeper techniques to speed up the process of tracking down when behavior is introduced"
+author: shiva
+image: /img/git-bisect/git-logo.png
+---
+It’s one of those nights- your project has been moving along at breakneck
+pace, with all of its contributors committing refactors, improvements, and
+bugfixes every day. You’ve been using git commands so much recently that you
+can rely on your fingers’s muscle memory to do all of your git operations for
+you. All of a sudden, you get that dreaded message from a teammate:
+
+> The firmware flasher is broken, and we don’t know when it broke
+
+If you’re anything like me, your reaction looks something like this.
+
+| ![](/img/git-bisect/gambino.gif) |
```

Comment:
This was the best I could do for a picture with a decent-looking caption, but suggestions are welcome. Maybe center it?